src/main/java/org/geysermc/resources/state/type/wall/ConnectionWestMapper.java

```diff
@@ -0,0 +1,15 @@
+package org.geysermc.resources.state.type.wall;
+
+import org.geysermc.resources.Pair;
+import org.geysermc.resources.state.StateMapper;
+import org.geysermc.resources.state.StateRemapper;
+
+@StateRemapper(value = "west", blockRegex = ".*_wall.?$")
+public class ConnectionWestMapper extends StateMapper<String> {

```

Comment:
Needs to be named to WallWestMapper. Naming convention is `(block)(java state)Mapper`.